src/index.html

```diff
@@ -2,17 +2,99 @@
+         aria-label="4 stars"
+        data-value="4">
+          <svg xmlns="http://www.w3.org/2000/svg" width="13"
+           height="12" fill="none"
+           viewBox="0 0 13 12">
+            <path fill="#E5E5E5"
+              d="M5.792.356a.515.515 0 0 1 .98 0l1.13 3.477a.515.515 0 0 0 .49.356h3.656c.499 0 .707.639.303.932l-2.958 2.15a.515.515 0 0 0-.187.575l1.13 3.477a.515.515 0 0 1-.793.576L6.585 9.75a.515.515 0 0 0-.606 0L3.022 11.9a.515.515 0 0 1-.793-.577l1.13-3.477a.515.515 0 0 0-.188-.576L.213 5.121a.515.515 0 0 1 .303-.932h3.656a.515.515 0 0 0 .49-.356L5.792.356z" />
+          </svg>
+        </a>
+        <a href="#" class="card__star card__star--3"
+         aria-label="3 stars"
+        data-value="3">
+          <svg xmlns="http://www.w3.org/2000/svg" width="13"
+          height="12" fill="none"
+          viewBox="0 0 13 12">
+            <path fill="#E5E5E5"
+              d="M5.792.356a.515.515 0 0 1 .98 0l1.13 3.477a.515.515 0 0 0 .49.356h3.656c.499 0 .707.639.303.932l-2.958 2.15a.515.515 0 0 0-.187.575l1.13 3.477a.515.515 0 0 1-.793.576L6.585 9.75a.515.515 0 0 0-.606 0L3.022 11.9a.515.515 0 0 1-.793-.577l1.13-3.477a.515.515 0 0 0-.188-.576L.213 5.121a.515.515 0 0 1 .303-.932h3.656a.515.515 0 0 0 .49-.356L5.792.356z" />
+          </svg>
+        </a>
+        <a href="#" class="card__star card__star--2"
+         aria-label="2 stars"
+        data-value="2">
+          <svg xmlns="http://www.w3.org/2000/svg" width="13"
+          height="12" fill="none"
+          viewBox="0 0 13 12">
+            <path fill="#E5E5E5"
+              d="M5.792.356a.515.515 0 0 1 .98 0l1.13 3.477a.515.515 0 0 0 .49.356h3.656c.499 0 .707.639.303.932l-2.958 2.15a.515.515 0 0 0-.187.575l1.13 3.477a.515.515 0 0 1-.793.576L6.585 9.75a.515.515 0 0 0-.606 0L3.022 11.9a.515.515 0 0 1-.793-.577l1.13-3.477a.515.515 0 0 0-.188-.576L.213 5.121a.515.515 0 0 1 .303-.932h3.656a.515.515 0 0 0 .49-.356L5.792.356z" />
+          </svg>
+        </a>
+        <a href="#" class="card__star card__star--1"
+         aria-label="1 stars"
+        data-value="1">
+          <svg xmlns="http://www.w3.org/2000/svg" width="13"
+           height="12" fill="none"
+           viewBox="0 0 13 12">
+            <path fill="#E5E5E5"
+              d="M5.792.356a.515.515 0 0 1 .98 0l1.13 3.477a.515.515 0 0 0 .49.356h3.656c.499 0 .707.639.303.932l-2.958 2.15a.515.515 0 0 0-.187.575l1.13 3.477a.515.515 0 0 1-.793.576L6.585 9.75a.515.515 0 0 0-.606 0L3.022 11.9a.515.515 0 0 1-.793-.577l1.13-3.477a.515.515 0 0 0-.188-.576L.213 5.121a.515.515 0 0 1 .303-.932h3.656a.515.515 0 0 0 .49-.356L5.792.356z" />
+          </svg>
+        </a>
+        <h3 class="card__reviews">Reviews: 5</h3>
+      </div>
+
+      <h3 class="card__price">Price: $2.199</h3>
+    </div>
+    <div class="card__actions">
+      <button type="submit" class="card__button"
+      data-qa="hover">
+            BUY
+      </button>
+    </div>
+       </div>
```

Comment:
There is an extra closing `div` tag on line 98 that does not match any opening tag. This could cause rendering issues.